dm/worker/server.go

```diff
@@ -668,7 +667,7 @@ func (s *Server) startWorker(cfg *config.MysqlConfig) error {
 
 	subTaskCfgs := make([]*config.SubTaskConfig, 0, 3)
 
-	ectx, cancel := context.WithTimeout(s.etcdClient.Ctx(), time.Second*3)
+	ectx, cancel := context.WithTimeout(s.etcdClient.Ctx(), time.Second*5)
```

Comment:
How about giving it another name because it's also used for database operation now (no only for **e**tcd)?

Reply:
I create a new ctx for getting min position